source/utilities.tex

```diff
@@ -2840,10 +2840,10 @@
     get(tuple<Types...>& t) noexcept;
 template<size_t I, class... Types>
   constexpr tuple_element_t<I, tuple<Types...>>&&
-    get(tuple<Types...>&& t) noexcept;        // Note A
+    get(tuple<Types...>&& t) noexcept;        // see Note 1
```

Comment:
There's a small risk that we'll insert another note and "Note 1" would have to be changed to "Note 2" but that seems acceptable.

The notes are just restating how references work, so aren't adding anything normative. They're fine to become `note` environments.

Reply:
OK, sure. This is a bit of a novel thing to keep track of, but this seems mangeable.

Maybe an alternative would be to say something like `// #1`, and then say "For the overload marked `#1`" in the note. Thoughts? @jwakely, @jensmaurer

Reply:
Works for me

Reply:
We could even just change the notes to say "for the second overload" and "for the third overload"

Reply:
Or we could strike the notes, and add an example that demonstrates the properties explained in the notes. Let me know if you want me to prepare such an example for the WD. For the ISO C++23 doc, any solution is fine with me.

Reply:
Yes, please!

Reply:
I'll apply this PR to the WD in any case, but please do go ahead with a better change.